### PR TITLE
add src repos delete command

### PR DESCRIPTION
**Pull request by ggilmore**
`src repos delete` allows you to delete one or more repositories from your sourcegraph instance. 

@slimsag I will revert my changes that add `multierrror` and `pkg/errors`. I had already written the file using those packages, so I thought I'd throw up a PR first.

### CODE REVIEW

File: cmd/src/repos_delete.go [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+		repoID, err := fetchRepositoryID(repoName)
+		if err != nil {
+			return err
+		}
+
+		query := `mutation DeleteRepository($repoID: ID!){
+		deleteRepository(repository: $repoID) {
+			alwaysNil
+		}
+	}`
```
Comment (slimsag) [inline cmd/src/repos_delete.go]: Formatting / indention here seems wonky

File: cmd/src/repos_delete.go [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+		flagSet.Parse(args)
+		var errs *multierror.Error
+		for _, repoName := range flagSet.Args() {
+			err := deleteRepository(repoName)
+			if err != nil {
+				err = errors.Wrapf(err, "Failed to delete repository %q", repoName)
+				errs = multierror.Append(errs, err)
+			}
+		}
+		return errs.ErrorOrNil()
```
Comment (slimsag) [inline cmd/src/repos_delete.go]: I am fine with using multierror / pkg/errors here. I didn't realize your change was so small in this regard.

But for consistency sake, could you update the `repos_disable_enable.go` file to also use this approach?

**Comment by ggilmore** [comment]
@slimsag PTAL, updated

File: cmd/src/repos_delete.go [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+		repoID, err := fetchRepositoryID(repoName)
+		if err != nil {
+			return err
+		}
+
+		query := `mutation DeleteRepository($repoID: ID!){
+   deleteRepository(repository: $repoID) {
+	   alwaysNil
+    }
+}`
```
[the remark this answers is not in the document]
Comment (slimsag) [reply cmd/src/repos_delete.go]: It is not. :(

**Pull request merged**